payavel/orchestration
# [2.x] Use Laravel Prompts by Jess Archer

### **What does this PR do?** :robot:
- Uses `laravel/prompts` to beautify the commands.
- Drops support for Laravel 9.x and php 8.0.
- Rewords the commands.
- Persists name into the application.

### **How should this be tested?** :microscope:
Run the existing commands in your console and make sure they are beautiful and working as expected.

### **Any background context you would like to provide?** :construction:
Note, this should also be tested on windows (without WSL), as it doesn't support prompts, and it must fallback to the standard symphony prompts.

### **Does this relate to any issue?** :link:
Closes #12
